src/components/label/template.njk

```diff
@@ -9,7 +9,7 @@
     {{ params.hintHtml | safe if params.hintHtml else params.hintText }}
   </span>
   {% endif %}
-  {% if params.errorMessage %}
-  {{ govukErrorMessage(params.errorMessage) -}}
+  {% if params.error.active %}
+  {{ govukErrorMessage(params.error.message) -}}
```

Comment:
This is inconsistent with the way the errorMessage attributes are passed in the fieldset - they're not nested there.